website_apps_store/tests/test_tour_website_apps_store.py

```diff
@@ -11,14 +11,6 @@ def setUp(self):
         super(TestUi, self).setUp()
         self.tour = "odoo.__DEBUG__.services['web_tour.tour']"
 
-    def test_download_zip_tour(self):
```

Comment:
You probably want to do the following:

In `generate_zip_files` extract this in an other method name `_get_module_path`:

``` python
 module_path = product.module_path + '/'\
                + product.odoo_module_version_id.technical_name
```

to get it like that:

``` python
module_path = product._get_module_path()
```

Then you need to mock this method to return your special test path:

``` python
[...]
import mock

mock_get_module_path = (
    'openerp.addons.apps_download.models.product.ProductProduct._get_module_path'
) # it might be odoo.addons in v11 to check

[...]

    def test_download_zip_tour(self):
        with mock.patch(mock_get_module_path) as mock_func:
            mock_func.return_value = <your test path string>
            self.phantom_js('/shop', ...)
```